src/index.test.ts

```diff
@@ -51,38 +53,27 @@ describe('SPVWalletClient routing', () => {
-    ${'GetDestinations'}                          | ${'post'}   | ${'destination/search'}          | ${() => spvWalletClient.GetDestinations({}, {}, {})}
-    ${'GetDestinationsCount'}                     | ${'post'}   | ${'destination/count'}           | ${() => spvWalletClient.GetDestinationsCount({}, {})}
-    ${'NewDestination'}                           | ${'post'}   | ${'destination'}                 | ${() => spvWalletClient.NewDestination({})}
-    ${'UpdateDestinationMetadataByID'}            | ${'patch'}  | ${'destination'}                 | ${() => spvWalletClient.UpdateDestinationMetadataByID('', {})}
-    ${'UpdateDestinationMetadataByLockingScript'} | ${'patch'}  | ${'destination'}                 | ${() => spvWalletClient.UpdateDestinationMetadataByLockingScript('', {})}
-    ${'UpdateDestinationMetadataByAddress'}       | ${'patch'}  | ${'destination'}                 | ${() => spvWalletClient.UpdateDestinationMetadataByAddress('', {})}
-    ${'GetTransaction'}                           | ${'get'}    | ${'transaction?id='}             | ${() => spvWalletClient.GetTransaction('')}
-    ${'GetTransactions'}                          | ${'post'}   | ${'transaction/search'}          | ${() => spvWalletClient.GetTransactions({}, {}, {})}
-    ${'GetTransactionsCount'}                     | ${'post'}   | ${'transaction/count'}           | ${() => spvWalletClient.GetTransactionsCount({}, {})}
-    ${'DraftTransaction'}                         | ${'post'}   | ${'transaction'}                 | ${() => spvWalletClient.DraftTransaction({} as TransactionConfigInput, {})}
-    ${'DraftToRecipients'}                        | ${'post'}   | ${'transaction'}                 | ${() => spvWalletClient.DraftToRecipients([] as Recipients, {})}
-    ${'RecordTransaction'}                        | ${'post'}   | ${'transaction/record'}          | ${() => spvWalletClient.RecordTransaction('', '', {})}
-    ${'UpdateTransactionMetadata'}                | ${'patch'}  | ${'transaction'}                 | ${() => spvWalletClient.UpdateTransactionMetadata('', {})}
-    ${'GetUtxo'}                                  | ${'get'}    | ${'utxo?tx_id=&output_index=0'}  | ${() => spvWalletClient.GetUtxo('', 0)}
-    ${'GetUtxos'}                                 | ${'post'}   | ${'utxo/search'}                 | ${() => spvWalletClient.GetUtxos({}, {}, {})}
-    ${'GetUtxosCount'}                            | ${'post'}   | ${'utxo/count'}                  | ${() => spvWalletClient.GetUtxosCount({}, {})}
-    ${'GetSharedConfig'}                          | ${'get'}    | ${'shared-config'}               | ${() => spvWalletClient.GetSharedConfig()}
+    spvWalletMethod                               | httpMethod  | path                                       | act
+    ${'GetUserInfo'}                              | ${'get'}    | ${'users/current'}                         | ${() => spvWalletClient.GetUserInfo()}
```

Comment:
It would be awesome to have a tests with mocked spv-wallet server and mocked responses to see how changed methods are really working

Reply:
Will try 🙈

Reply:
We decided to move it to separated task -> it will be: SPV-1087